demo-notebooks/testing_instructions.txt

```diff
@@ -0,0 +1,40 @@
+A tested version of Search & Download has been created, named "Main_Site_Search_&_Download-Python-new-testable-example.ipynb"
+Modifications were made to handle our example dataset and improve error messages, but the core features of the notebook remain unchanged. 
+
+After cloning the repository, navigate to the directory "demo-notebooks"
+
+You must first install Pytest and nbval, using:
+
+```pip install pytest```
+
+then 
+
+```pip install nbval```
+
+After confirming that the testing framework has been installed, you need to provide a valid token from your ESS-DIVE account (new one needed every 24 hours):
+
+- Open "secrets.txt": enter your authentication token and save the file 
+
+NOTE: Authentication tokens for the same account vary between the specific site used (e.g. ess-dive production site vs. data-stage)
+Please ensure that you are retrieving the token from the correct site.
+"Main_Site_Search_&_Download-Python-new-testable-example.ipynb" uses the token from the ess-dive production site
+"Data-Stage_Search_&_Download-Python-new-testable-example.ipynb" uses the token from data-stage.ess-dive.lbl.gov
```

Comment:
Remove line 21 if this file will not be included in the PR: "Data-Stage_Search_&_Download-Python-new-testable-example.ipynb"

Reply:
@mburrus These are great comments.  let's merge @franflame code as-is and then someone from our team can take if from there and make it ready for public use.

Reply:
@mburrus I take back my previous comment.  This is a great point.